src/models/sourcecodemodel.cpp

```diff
@@ -196,6 +199,5 @@ void SourceCodeModel::setCallerCalleeResults(const Data::CallerCalleeResults& re
 {
     beginResetModel();
```

Comment:
I guess this doesn't actually influence the model at all, right? so we should actually remove the `{begin,end}ResetModel` here now - only the calls to `SourceCodeModel::setDisassembly` have an effect. I'd actually go as far as merging the two methods into one, to ensure that one cannot set one without the other.

Reply:
Merged the two methods for both `SourceCodeModel` and `DisassemblyModel`. In case of `SourceCodeModel` we no longer need to copy `Data::CallerCalleeResults` after replacing
`auto entry = m_callerCalleeResults.entry(disassemblyOutput.symbol);`
with
`const auto entry = results.entries.find(disassemblyOutput.symbol);`.

The change here is that `entry` creates an empty symbol if it's not found and in this case we would quickly discard it since `entry->offsetMap` would be empty.